pkg/testcase/networkconnectivity.go

```diff
@@ -0,0 +1,82 @@
+package testcase
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+// TestInternodeConnectivityMixedOS Deploys services in the cluster and validates communication between linux and windows nodes
+func TestInternodeConnectivityMixedOS(deleteWorkload bool) {
+	_, err := shared.ManageWorkload("apply",
+		"pod_client.yaml","windows_app_deployment.yaml")
+	if err != nil {
+		GinkgoT().Errorf("error: %v", err)
```

Comment:
this func already return the test as failed. 
is similar to `	Expect(err).NotTo(HaveOccurred(), "xxxxxx  not deployed")`

so no need to return as well here

Reply:
Should generalize the approach then, having different approach is kind of confusing... so should I go with this or the Expect? Ideally we can go with Expect being this a test itself, wdyt?